pkg/networkservice/chains/nsmgr/server.go

```diff
@@ -83,6 +84,13 @@ type serverOptions struct {
 // Option modifies server option value
 type Option func(o *serverOptions)
 
+// WithDialTimeout sets gRPC Dial timeout for the server
+func WithDialTimeout(timeout time.Duration) Option {
```

Comment:
Can we create an option for connect paramaeters?
```suggestion
func WithConnectOptions(options...connect.Option)Option {
```

Reply:
It would make `nsmgr` user know about `connect` package + it would make common NSMgr creation (without dial timeout) more complex:
```go
nsmgr := nsmgr.NewServer(ctx, tokenGenerator,
    ...
    nsmgr.WithConnectOptions(
        connect.WithDialOptions(...)),
)
```

Reply:
In general, It solves the problem with duplicating options from `connnect` in `nsmgr`

Reply:
Done.